pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -61,13 +64,21 @@ stdenv.mkDerivation rec {
 
     substituteInPlace make.tmpl.in --replace "@systemdsystemunitdir@" "$out/lib/systemd/system"
     substituteInPlace libdm/make.tmpl.in --replace "@systemdsystemunitdir@" "$out/lib/systemd/system"
+
+    substituteInPlace scripts/blkdeactivate.sh.in \
+      --replace '/bin/mountpoint' '${util-linux}/bin/mountpoint' \
+      --replace '/bin/umount' '${util-linux}/bin/umount' \
+      --replace '/bin/lsblk' '${util-linux}/bin/lsblk' \
+      --replace '/bin/findmnt' '${util-linux}/bin/findmnt' \
+      --replace '/bin/sort' '${coreutils}/bin/sort' \
```

Comment:
`blkdeactivate` gets invoked by users, not scripts, right? Can't we be semi-sure that `util-linux` and `coreutils` are in their paths (due to `nixos/modules/config/system-path.nix`).

Reply:
It gets invoked by `blk-availability.service` on shutdown, if we choose to enable that.